conda_forge_tick/auto_tick.py

```diff
@@ -965,6 +966,7 @@ def initialize_migrators(
             PipWheelMigrator(),
             MPIPinRunAsBuildCleanup(),
             DependencyUpdateMigrator(python_nodes),
+            QtQtMainMigrator(),
```

Comment:
This should be added to the MigrationYaml migrator, not VersionMigrator

Reply:
Thank you, i was unsure of this one